README.md

```diff
@@ -30,12 +30,20 @@ Be prepared to demonstrate your understanding of this week's concepts by answeri
 
 1. What is React JS and what problems does it solve? Support your answer with concepts introduced in class and from your personal research on the web.
 
+React Js is a library for building User Interface. It solves the problem of apps having to render at the same page so inorder to fix this problem we build different components so that it doesn't cause the page to reload everytime. And if we want to change specific page we can do it on different components. 
+
 1. Describe component state.
 
+Component State is the state that stores information or data and when state changes the component responds by re-rendering.
```

Comment:
Good job Bikesh.
Your description here, using the idea of a data store that can cause the DOM to re-render is very accurate.

React State is the current shape or type or value of a piece of data in an Application. Its the way something is at a specific point in time.